docker-for-windows/install-windows-home.md

```diff
@@ -0,0 +1,146 @@
+---
+description: How to install Docker Desktop on Windows 10 Home
+keywords: Windows 10 Home, Home, Windows, install, download, run, Docker, local
+title: Install Docker Desktop on Windows Home
+---
+
+You can now install Docker Desktop on Windows 10 Home machines. Docker Desktop for WSL 2 Windows Home is a full version of Docker Desktop for Linux container development.
```

Comment:
"Docker Desktop for WSL 2 Windows Home": is it a thing ?

Reply:
Fixed. PTAL.